### ISSUE TITLE
Language Filter - 3.3.2 Labels or Instructions - The dropdown field is missing a form label

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9962]
As a user reliant on the screen reader to interpret form fields, I expect all form fields to have accessible names which can be correctly interpreted by the screen reader to give the name and purpose of the field.

**Linked work item:** 
#9521 

**Expected Result:**
When the language dropdown field receives focus, the screen reader should be able to interpret the name and purpose of the field using an accessible label associated with the field in the html.

**Actual Result:**
The dropdown field does not have a form label, meaning the screen reader is not able tocorrectly interpret the name and purpose of the field and just reads the field as 'blank' on focus.

**Evidence:**

Use NVDA to replicate:

![Image](https://github.com/archesproject/arches/assets/77161085/e5df37b4-e46d-45af-ba78-93b408a4f6d0)

**Comment by johnatawnclementawn** [comment]
This issue is dependent on our select2/chosen replacement work.

Each item is dynamically created in encapsulated chosen code, which prevents us from directly adding aria-labels to the items

**Comment by apeters** [comment]
@imogenroberts this should be fixed in the latest

**Comment by imogenroberts** [comment]
RETEST: the language filter no longer has any missing form labels and is therefore read out correctly by the screen reader.

**Evidence:**
![Language Filter - 3 3 2 Labels or Instructions FIXED](https://github.com/archesproject/arches/assets/77161085/99bf87b4-287f-4d3f-b674-fe0f7433bd90)
